emscripten.py

```diff
@@ -667,6 +667,11 @@ def add_standard_wasm_imports(send_items_map):
       'store_val_f64',
     ]
 
+  if settings.SPLIT_MODULE and settings.ASYNCIFY == 2:
+    # Calls to this function are generated by binaryen so it must be manually
+    # imported.
+    send_items_map['__load_secondary_module'] = '_emscripten_load_secondary_module'
```

Comment:
Why have give this function the same name within emscripten?  Then you can just do `extra_sent_items.append`

Reply:
All the other library functions in library_async.js have an emscripten prefix. Fine to change it match the binaryen name though. Preference?

Reply:
I think in this case since we already have the double leading underscores, and the symbols are clearly the same they should probably be named identically.